lib/jekyll/utils/rouge.rb

```diff
@@ -4,16 +4,16 @@ module Rouge
 
       def self.html_formatter(*args)
         Jekyll::External.require_with_graceful_fail("rouge")
-        html_formatter = ::Rouge::Formatters::HTML.new(*args)
-        return html_formatter if old_api?
-
-        ::Rouge::Formatters::HTMLPygments.new(html_formatter)
+        if old_api?
+          ::Rouge::Formatters::HTML.new(*args)
+        else
+          ::Rouge::Formatters::HTMLLegacy.new(*args)
```

Comment:
[Here](https://github.com/jneen/rouge#formatters) are all default formatters listed.  IMHO line numbers (which we rely on in a couple of tests) would also be present in `Rouge::Formatters::HTMLTable` and `Rouge::Formatters::HTMLLinewise` but `HTMLLegacy` should be the easiest update.